src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -641,6 +686,9 @@
 -   name: Trigger a task.
     text: >
         az acr task run -n MyTask -r MyRegistry
+-   name: Manually trigger a task that might otherwise be waiting for git commits or base image update triggers.
+    text: az acr task run --registry MyRegistry --name MyTask
```

Comment:
Same parameters as existing example. Description might be better though someone from ACR would need to validate that.